main/utils.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+
+	"github.com/Worldcoin/hubble-commander/bls"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func newWallet(ctx *cli.Context) error {
+	privateKey := make([]byte, 32)
+	_, err := rand.Read(privateKey)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	wallet, err := bls.NewWallet(privateKey, bls.Domain{0x00, 0x00, 0x00, 0x00})
```

Comment:
Is it correct to hardcode the domain to all zeros?

Reply:
I believe it is, looking through [the source](https://github.com/kilic/bn254/blob/master/bls/bls.go) the domain is only used when generating and verifying signatures, the domain isn't used when deriving the public key, which is all this script does.